jplag.frontend.golang/src/main/java/de/jplag/golang/Language.java

```diff
@@ -0,0 +1,69 @@
+package de.jplag.golang;
+
+import java.io.File;
+
+import de.jplag.ErrorConsumer;
+import de.jplag.TokenList;
+
+public class Language implements de.jplag.Language {
+
+    private static final String SHORT_NAME = "Go";
+    private static final String NAME = "Go Parser";
```

Comment:
Order would be better to put the long name before the short. Simply because of the readability but is not a must